activerecord/CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
-Adds support for multiple databases to `rails db:schema:cache:dump` and `rails db:schema:cache:clear`.
+*   Defining an Enum as a Hash with blank key, or as an Array with a blank a value, now raises an `ArgumentError`.
```

Comment:
Typo?
```suggestion
*   Defining an Enum as a Hash with blank key, or as an Array with a blank value, now raises an `ArgumentError`.
```

Reply:
Good catch, I missed the extra `a` here. Please fix this.

Reply:
done!